lib/ui/painting.dart

```diff
@@ -1566,27 +1566,72 @@ enum PixelFormat {
 /// To draw an [Image], use one of the methods on the [Canvas] class, such as
 /// [Canvas.drawImage].
 ///
+/// A class or method that receives an image object must call [dispose] on the
+/// handle when it is no longer needed. To create a shareable reference to the
+/// underlying image, call [clone]. The method or object that recieves
+/// the new instance will then be responsible for disposing it, and the
+/// underlying image itself will be disposed when all outstanding handles are
+/// disposed.
+///
+/// If `dart:ui` passes an `Image` object and the recipient wishes to share
+/// that handle with other callers, it is critical that [clone] is called
+/// _before_ [dispose]. A handle that has been disposed cannot create new
```

Comment:
maybe just simplify: "If `dart:ui` passes an `Image` object and the recipient wishes to share that handle with other callers, [clone] must be called _before_ [dispose]."

(The "is is critical that" sounds like empty prose)

Reply:
Done